man/systemd.network.xml

```diff
@@ -210,6 +212,15 @@
           <para>Identity Association Identifier for the interface, a 32-bit unsigned integer.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+      <listitem>
+        <para>Note that if any interface where no static IPv6 address is configured, and
+        neither DHCPv6 nor IPv6LL is enabled, shall be considered an IPv6 less interface.
+        IPv6 will be automatically disabled for that interface by writing "1" to
+        <filename>/proc/sys/net/ipv6/conf/<replaceable>ifname</replaceable>/disable_ipv6.</filename>
```

Comment:
"Note that an interface without any static IPv6 addresses configured, and neither DHCPv6 nor IPv6LL enabled, shall be considered to have no IPv6 support. IPv6 will be automatically disabled for that interface by writing "1" to ..."